lib/Pithub/Repos/Releases.pm

```diff
@@ -110,11 +110,57 @@ Examples:
             target_commitish => 'master',
             name             => 'v1.0.0',
             body             => 'Description of the release',
-            draft            => 0,
-            prerelease       => 0,
+            draft            => JSON::MaybeXS::false,           # or alternative below
+            prerelease       => JSON::MaybeXS::false,           # or alternative below
+            generate_release_notes => JSON::MaybeXS::false,     # or alternative below
         }
     );
 
+Booleans:
+
+Several of the attributes B<require> boolean values in the request that is sent
+to GitHub.  Zero (0) and one (1) are integers and will not be encoded correctly
+in the JSON encoded request.
+
+There are numerous options for your call to Pithub::Repos::Releases->create.
+
+=over
+
+=item JSON::MaybeXS
+
+Add the following to your code before the call to Pithub::Repos::Releases->create.
+
+    require JSON::MaybeXS;
+
+Then use the following values for the booleans:
+
+    JSON::MaybeXS::true
+    JSON::MaybeXS::false
```

Comment:
I wonder if we should change these to the function calls, since we know those will work more consistently?

Reply:
good idea - give me a few minutes

Reply:
@oalders I think I found all of them